cmd/crc/cmd/config/config.go

```diff
@@ -23,9 +23,10 @@ var (
 
```

Comment:
`5 files changed, 120 insertions(+), 19 deletions(-)`
Better to have more than one line/60 chars to describe such changes :) If something is listed in "List main as well as consequential changes you introduced or had to introduce." in the PR description, it needs to appear in the commit log as well.

Reply:
@cfergeau we are just adding a feature, I am not sure what I can add as part of description :confused:

Reply:
what it does, how to use it, ... (which reminds me this PR is missing a docs change)

Reply:
@cfergeau yes, we need to add the doc for it.